google/cloud/accesscontextmanager/doc/main.dox

```diff
@@ -0,0 +1,100 @@
+
+@code{.sh}
+git clone https://github.com/googleapis/google-cloud-cpp.git
+cd google-cloud-cpp/google/cloud/accesscontextmanager/quickstart
+@endcode
+
+@par Example: Quickstart
+
+The following shows the code that you'll run in the
+`google/cloud/accesscontextmanager/quickstart/` directory,
+which should give you a taste of the Access Context Manager API C++ client library API.
+
+@include quickstart.cc
+
+## Environment Variables
+
+- `GOOGLE_CLOUD_CPP_ENABLE_TRACING=rpc` turns on tracing for most gRPC
+  calls. The library injects an additional Stub decorator that prints each gRPC
+  request and response.  Unless you have configured your own logging backend,
+  you should also set `GOOGLE_CLOUD_CPP_ENABLE_CLOG` to produce any output on
+  the program's console.
+
+- `GOOGLE_CLOUD_CPP_ENABLE_TRACING=rpc-streams` turns on tracing for streaming
+  gRPC calls, such as `Client::Read()`, `Client::ExecuteQuery()`, and
+  `Client::ProfileQuery()`. This can produce a lot of output, so use with
+  caution!
+
+- `GOOGLE_CLOUD_CPP_TRACING_OPTIONS=...` modifies the behavior of gRPC tracing,
+  including whether messages will be output on multiple lines, or whether
+  string/bytes fields will be truncated.
+
+- `GOOGLE_CLOUD_CPP_ENABLE_CLOG=yes` turns on logging in the library. Basically
+  the library always "logs" but the logging infrastructure has no backend to
+  actually print anything until the application sets a backend or it sets this
+  environment variable.
+
+### Error Handling
+
+[status-or-header]: https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/status_or.h
+
+This library never throws exceptions to signal error, but you can use exceptions
+to detect errors in the returned objects. In general, the library returns a
+[`StatusOr<T>`][status-or-header] if an error is possible. This is an "outcome"
+type, when the operation is successful a `StatusOr<T>` converts to `true` in
+boolean context (and its `.ok()` member function returns `true`), the
+application can then use `operator->` or `operator*` to access the `T` value.
+When the operation fails a `StatusOr<T>` converts to `false` (and `.ok()`
+returns `false`). It is undefined behavior to use the value in this case.
+
+If you prefer to use exceptions on error, you can use the `.value()` accessor.
+It will return the `T` value or throw on error.
+
+For operations that do not return a value the library simply returns
+`google::cloud::Status`.
+
+### Retry, Backoff, and Idempotency Policies.
+
+The library automatically retries requests that fail with transient errors, and
+uses [exponential backoff] to backoff between retries. Application developers
+can override the default policies.
+
+[cloud-service-docs]: https://cloud.google.com/accesscontextmanager
+[exponential backoff]: https://en.wikipedia.org/wiki/Exponential_backoff
+[gcs-quickstart]: https://cloud.google.com/accesscontextmanager/docs/quickstarts 'Quickstarts'
```

Comment:
link is broken, but we actually don't use `gcs-quickstart` anywhere.... we should remove from the scaffolding.

Reply:
Removed. I am sending a separate PR to fix the generator.